src/workspace.js

```diff
@@ -422,6 +423,17 @@ module.exports = class Workspace extends Model {
       this.didChangeActivePaneItem(item)
       this.emitter.emit('did-change-active-pane-item', item)
     }
+
+    if (paneContainer === this.getCenter()) {
+      const activeTextEditorChanged =
+        (item instanceof TextEditor) || (this.activeTextEditor instanceof TextEditor)
```

Comment:
I think it'd be nice to consolidate these `item instanceof TextEditor` checks into one. Also, I would think we would not need to check `this.activeTextEditor instanceof TextEditor`, because `this.activeTextEditor` is always either a text editor or null.